Apps/WebClient/src/ClientApp/src/views/RegistrationView.vue

```diff
@@ -331,11 +356,11 @@ export default class RegistrationView extends Vue {
                         <b-form-input
                             id="emailInput"
                             v-model="$v.email.$model"
-                            data-testid="emailInput"
-                            type="email"
-                            placeholder="Your email address"
                             :disabled="isPredefinedEmail || !isEmailChecked"
                             :state="isValid($v.email)"
+                            data-testid="emailInput"
+                            placeholder="Your email address"
+                            type="email"
```

Comment:
Was this a manual change or is your formatter automatically reordering attributes alphabetically?

Reply:
it was automatic